src/helpers/asHTML.ts

```diff
@@ -203,41 +203,86 @@ const wrapMapSerializerWithStringChildren = (
 	return wrapMapSerializer(modifiedMapSerializer);
 };
 
+/**
+ * Configuration that determines the output of `asHTML()`.
+ */
+type AsHTMLConfig = {
+	/**
+	 * An optional link resolver function to resolve links.
+	 * Without it you're expected to use the `routes` options from the API.
+	 */
+	linkResolver?: LinkResolverFunction<string | null | undefined> | null;
```

Comment:
I believe we can remove the `<string | null | undefined>` as that is now the default. Could you confirm?

Reply:
Indeed!